lib/main_publiccloud.pm

```diff
@@ -123,6 +125,21 @@ sub load_latest_publiccloud_tests {
             loadtest "xfstests/generate_report";
         } elsif (get_var('PUBLIC_CLOUD_AZURE_NFS_TEST')) {
             loadtest("publiccloud/azure_nfs", run_args => $args);
+        } elsif (get_var('PUBLIC_CLOUD_SLEM_TESTS')) {
+            loadtest("transactional/enable_selinux") if (get_var('ENABLE_SELINUX'));
+            loadtest("containers/k3s_cli_check") if (get_var('SLE_MICRO_K3S'));
```

Comment:
```suggestion
            loadtest("containers/k3s_cli_check") if (get_var('PUBLICCLOUD_SLEM_K3S'));
```

at least for me this would keep common logic within all variables